CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/DateTimeRangeInputPane.java

```diff
@@ -603,8 +604,7 @@ private static ObservableList<String> getZoneIds() {
         final DateTimeFormatter zf = DateTimeFormatter.ofPattern("Z");
         final Instant instant = Instant.now();
         final Set<String> zoneSet = ZoneId.getAvailableZoneIds();
-        final List<ZoneId> zoned = new ArrayList<>();
-        zoneSet.stream().map(ZoneId::of).forEach(zoned::add);
+        final List<ZoneId> zoned = zoneSet.stream().map(ZoneId::of).collect(Collectors.toList());
```

Comment:
the `.collect` can be made `.toList`

Reply:
Turns out changing the code from `.collect(Collectors.toList());` to `.toList();` breaks the data access view. Sonar recommends  `.collect(Collectors.toList());` so I'm going to stick with that.

Reply:
Yep I've realised that `.toList()` is actually more equivalent to `.collect(Collectors.toUnmodifiableList())` and so while there are a lot of instances where we can make the switch, there are some instances such as this one where that won't be the case